src/main/resources/templates/gardens/add.html

```diff
@@ -38,6 +38,7 @@
     <div class="form-group">
         <label th:for="owner">Garden Owner</label>
         <select name="owner">
+            <<option>--Please select one--</option>
```

Comment:
It looks like there is an extra less than carrot here "<". Is that extra or is it needed for something?